### PR TITLE
Add Docker and gulp config files

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+django:
+  build: .
+  command: >
+    /bin/bash -c "
+    pip install -r requirements/dev.txt &&
+    python manage.py runserver 0.0.0.0:8002"
```
Comment (ushkarev) [inline docker-compose.yml]: should probably be 8004

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  volumes:
+    - .:/app
+    - /tmp/pip_download_cache/:/tmp/pip_download_cache
+  ports:
+    - "8002:8002"
```
Comment (ushkarev) [inline docker-compose.yml]: should probably be 8004

**Pull request merged**